app/models/user.rb

```diff
@@ -18,6 +18,7 @@ class User < ApplicationRecord
   has_one :google_credential, -> { type_is("GoogleCredential") }, class_name: "Credential", inverse_of: :user
   has_one :gmail_credential, -> { type_is("GmailCredential") }, class_name: "Credential", inverse_of: :user
   has_one :google_tasks_credential, -> { type_is("GoogleTasksCredential") }, class_name: "Credential", inverse_of: :user
+  has_one :microsoft_graph_credential, -> { type_is("MicrosoftGraphCredential") }, class_name: "Credential", inverse_of: :user
```

Comment:
I’ve been adding simple tests for these associations. To be consistent with the current test suite, I believe that `models/user_test` should get a test for user.microsoft_graph_credential. Also, let’s add a microsoft_graph_credential_test.rb file (you can look at the google one).

Reply:
Thanks, done.